docs/docs/tutorials/basic_arithmetic_grpc_server.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+title: Creating a Basic Arithmetic gRPC Server
+parent: Tutorials
+nav_order: 3
+---
+
+## Creating a Basic Arithmetic gRPC Server
+
+First look at the `main.go` file of Arithmetic server that will serve on a given port and listening at the endpoint we defined in proto files.
+
+At the top of the file, we define the handlers for our gRPC methods.
+
+Handlers are:
+
+```go
+// This is the struct that we will implement all the handlers on
+type Server struct {
+	a      *pbExample.Request
+	b      *pbExample.Request
+	Result *pbExample.Response
+}
+
+// This is where you implement the handler for the Add method
+func (s *Server) Add(ctx context.Context, request *pbExample.Request) (*pbExample.Response, error) {
+	a, b := request.GetA(), request.GetB()
+
+	result := a + b
+
+	return &pbExample.Response{Result: result}, nil
+}
+
+// This is where you implement the handler for the Divide method
+func (s *Server) Divide(ctx context.Context, request *pbExample.Request) (*pbExample.Response, error) {
+	a, b := request.GetA(), request.GetB()
+
+	result := a / b
+
+	return &pbExample.Response{Result: result}, nil
+}
+
+// This is where you implement the handler for the Multiply method
+func (s *Server) Multiply(ctx context.Context, request *pbExample.Request) (*pbExample.Response, error) {
+	a, b := request.GetA(), request.GetB()
+
+	result := a * b
+
+	return &pbExample.Response{Result: result}, nil
+}
+
+// This is where you implement the handler for the Subtract method
+func (s *Server) Subtract(ctx context.Context, request *pbExample.Request) (*pbExample.Response, error) {
+	a, b := request.GetA(), request.GetB()
+
+	result := a - b
+
+	return &pbExample.Response{Result: result}, nil
+}
```

Comment:
This feels like it might just be padding the content of this file. We can include the implementation of `Add`, but the rest of this seems excessive to me, we can always link to the example implementation.